core/templates/dev/head/editor/exploration_feedback.html

```diff
@@ -39,7 +39,7 @@
             <div><[t.summary]></div>
           </td>
           <td class="col-lg-3 col-md-3 col-sm-3">
-            <span ng-if="t.original_author_username"><[t.original_author_username]></span>
+            <span ng-if="t.original_author_username"><profile-link-text username="<[t.original_author_username]>"></profile-link-text></span>
```

Comment:
Actually -- no need to interpolate. Prefer profile-link-text username="t.original_author_username", and change '@' to '&' in the directive. Ditto elsewhere.